tests/test_utils.py

```diff
@@ -86,6 +86,9 @@ def assert_frames_close(actual, expected, **kwargs):
         'instead found' + expected.index.values + 'vs' + actual.index.values
 
     for col in expected.columns:
+        # if for Vensim outputs where constant values are only in the first row
```

Comment:
Can you create an issue in the test-models repo discussing this change, and how some of the tests now have only the top line fully populated? Would be good to document that difference there and check in with the other users of the repo that they are ok with that. =)

Reply:
Sure